docs/src/user_docs/assembly/io_operations.md

```diff
@@ -3,7 +3,7 @@ Miden assembly provides a set of instructions for moving data between the stack
 
 * **Program code**: values to be moved onto the stack can be hard-coded in a program's source code.
 * **Environment**: values can be moved onto the stack from environment variables. Currently, the available environment variables are *stack_depth*, which holds the current depth of the stack, and *local_address*, which stores absolute addresses of local variables. In the future, other environment variables may be added.
-* **Advice tape**: values can be moved onto the stack from a non-deterministic advice tape contained within the advice provider. Values are always read from the head of the advice tape, and once a value is read, it is removed from the tape. There is no limit on the number of values in the advice tape.
+* **Advice stack**: values can be moved onto the stack from a non-deterministic advice stack contained within the advice provider. Values are always read from the head of the advice stack, and once a value is read, it is removed from the stack. There is no limit on the number of values in the advice stack.
```

Comment:
Let's change this to:

**Advice stack**: values can be moved onto the stack from the advice stack contained within the advice provider (i.e., by popping the advice stack). There is no limit on the number of values in the advice stack.